src/tools/genpolicy/src/settings.rs

```diff
@@ -86,4 +86,17 @@ impl Settings {
             &self.other_container
         }
     }
+
+    pub fn panic_on_undefined_variables(&self, var_name: &str) {
+        if !self
+            .request_defaults
+            .CreateContainerRequest
+            .allow_env_regex_map
+            .contains_key(var_name)
+        {
+            panic!(
```

Comment:
Since we are removing the rule that "allows for anything", panic, since the policy we would produce otherwise won't be allowed anyways